src/System.Text.Encoding.Extensions/Encoding.Extensions.Tests.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.22609.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Encoding.Extensions.Tests", "tests\4.0.10.0\Encoding\Encoding.Extensions.Tests.csproj", "{9574CEEC-5554-411B-B44C-6CA9EC1CEB08}"
```

Comment:
This is the first time I've seen us using this directory structure for tests on GitHub, i.e. "tests\4.0.10.0\"... is this intentional?

Also, shouldn't the project be named System.Encoding.Extensions.Tests, rather than just Encoding.Extensions.Tests.